che-theia-plugins.yaml

```diff
@@ -562,3 +562,15 @@ plugins:
       revision: 1e15d3495da89072d48cf583d48d92829f0c4b82
     extensions:
       - https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-eslint/vscode-eslint-2.1.1-1e15d3.vsix
+  - repository:
+      url: https://github.com/gabrielbb/vscode-lombok
+      revision: master
+    sidecar:
+      directory: java8
+      name: vscode-lombok
```

Comment:
does it works with java (java11) ? As it's the latest lts

Reply:
yes and I made the corresponding changes to demo, devfile and extension config